src/components/Visualizations/Graph/Graph.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import c3 from 'c3';
+import uuid from 'node-uuid';
+import npmColor from 'color';
+
+const Graph = ({ id, height = 320 }) => (
+  <div style={{ height }} id={id} />
+);
+
+const generateGraph = ({ columns, type, name, height = 320 }, id) => {
+  if (columns && columns.length > 0) {
+    const columnVals = columns.map(column => column.games);
+    const xVals = columns.map(column => column.x);
+    c3.generate({
+      bindto: `#${id}`,
+      data: {
+        x: 'x',
+        columns: [
+          ['x', ...xVals],
+          [name, ...columnVals],
+        ],
+        color: (color, data) => {
+          if (data.index || data.index === 0) {
+            const { index, value } = data;
+            const wins = columns[index] && columns[index].win;
+            if (!value) {
+              return npmColor().rgb(255, 255, 255);
+            }
+            const percent = wins / value;
+            const adjustedVal = percent >= 0.5 ?
```

Comment:
This is just changing the color scaling to be non-linear. The reason I check 0.5 is because otherwise, 50% will appear green vs. yellow due to this scaling. I'll add a comment.